openqa-trigger/schedule-obs.rb

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/ruby
+require 'dotenv/load'
+require 'pry'
+require 'faraday'
+require 'xmlhash'
```

Comment:
We should avoid using `xmlhash` and instead rely on `Nokogiri`: https://github.com/openSUSE/open-build-service/issues/10085